src/main/java/org/broadinstitute/hellbender/utils/variant/GATKVCFHeaderLines.java

```diff
@@ -60,6 +59,9 @@ private static void addFilterLine(final VCFFilterHeaderLine line) {
         addFilterLine(new VCFFilterHeaderLine(READ_POSITION_FILTER_NAME, "median distance of alt variants from end of reads"));
         addFilterLine(new VCFFilterHeaderLine(CONTAMINATION_FILTER_NAME, "contamination"));
         addFilterLine(new VCFFilterHeaderLine(DUPLICATED_EVIDENCE_FILTER_NAME, "evidence for alt allele is overrepresented by apparent duplicates"));
+        addFilterLine(new VCFFilterHeaderLine(F1R2_ARTIFACT_FILTER_NAME, "over-representation of F1R2 reads among alt"));
```

Comment:
I would say that it's an F1R2 orientation bias artifact instead, because just saying over-representation could be interpreted to mean that you *just* look at F1R2 and F2R1 counts without learning priors.  We don't want users to jump to bad conclusions about the model.

Reply:
done